src/layout/header/Menu.tsx

```diff
@@ -1,33 +1,32 @@
-import React, { useState } from 'react';
+import React, { useState, MouseEvent } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
 import styled from 'styled-components';
-import { Button, Menu as MuiMenu, MenuItem, Typography, IconButton } from '@material-ui/core';
+import { Button, Menu as MuiMenu, MenuItem, Typography, IconButton, Divider } from '@material-ui/core';
 import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
 import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
 import AccountCircle from '@material-ui/icons/AccountCircle';
 import { useSelector } from 'react-redux';
 import { RootStore } from '../../redux/reducers/rootReducer';
-import NormalText from '../../components/NormalText';
 import { UrlPathTemplate } from '../../utils/urlPaths';
 
 const StyledMenu = styled.div`
   grid-area: menu;
 `;
 
-const StyledAdminMenu = styled.div`
-  border-bottom: 1px solid ${({ theme }) => theme.palette.box.main};
+const StyledPaddedTypography = styled(Typography)`
+  padding-left: 1rem;
+  padding-right: 1rem;
 `;
```

Comment:
bruker `disableGutters` for hvert `MenuItem` og setter padding manuelt her for å få til skikkelig like innrykk for alle elementer og unngå dette: 
![bilde](https://user-images.githubusercontent.com/6313468/104312599-1f042800-54d7-11eb-8a4e-18327a0d0b66.png)

Alternativt kan vi bruke `<MenuItem disabled>` for det som skal være heading i menyen, men da vil teksten bli grå, som bare kan overstyres med å sette `opacity: 1 !important`.  Det er kanskje en enklere løsning, men vi må evt leve med en `!important` da...

Reply:
tror ikke dette er så farlig egentlig. tipper vi kommer til å endre på menyen (og de overskriftene) etterhvert også.